src/breadcrumb-group/__tests__/breadcrumb-group.test.tsx

```diff
@@ -136,7 +141,12 @@ describe('BreadcrumbGroup Component', () => {
     });
 
     test('does not throw an error when a safe javascript: URL is passed', () => {
-      const element = renderBreadcrumbGroup({ items: [{ text: '', href: 'javascript:void(0)' }] });
+      const element = renderBreadcrumbGroup({
+        items: [
+          { text: '1', href: 'javascript:void(0)' },
+          { text: '2', href: 'javascript:void(0)' },
```

Comment:
Why add second item here?

Reply:
Hi @jorycunningham why do we change here?

Reply:
I did that because the behavior of the last in the series in different in regards to href and being disabled